### DISCUSSION TITLE
Unable to get Session replays work

### DISCUSSION THREAD

**Discussion by uhlhosting** [stayallive/wp-sentry#162] [Q&A] [answered]
I have according to the WP UI in Settings all in place:

<img width="628" alt="image" src="https://github.com/stayallive/wp-sentry/assets/6324047/13094980-1e76-43c5-b81b-5c818b82b65e">

<img width="621" alt="image" src="https://github.com/stayallive/wp-sentry/assets/6324047/2f6731ce-0853-4451-87d3-ffdc97ed3f0c">

Yet on Sentry side, a bit different variables are shown as needed.

**Comment by stayallive** [comment]
Check out the configuration documentation: https://github.com/stayallive/wp-sentry#configuration

If you scroll a bit you will get to the "(Optionally) enable JavaScript Session Replay" section and that should explain how to set all those values!

**Reply by uhlhosting** [reply]
@stayallive already double checked the readme, and applied all settings, still there is no sessions.

**Reply by stayallive** [reply]
Are you able to share the site URL so I can check if I see anything amiss? If you want to do it privately my e-mail is listed on my GitHub profile.

**Reply by uhlhosting** [reply]
Sure, https://eofs.dev . Let me know if you need i could share also wp-config by mail if you need it.

**Reply by uhlhosting** [reply]
@stayallive any updates?

**Reply by stayallive** [reply]
@uhlhosting I'm very sorry, this totally slipped my mind... but I did just now take a look and my session did sent a Replay session to your self-hosted Sentry instance. So I think everything on this plugin side works but maybe your self-hosted instance is unable to process/display the Replay sessions? You could try to send it temporarily to a sentry.io DSN to make sure that is the case.

![image](https://github.com/stayallive/wp-sentry/assets/1090754/f9958c1a-0e37-4efa-8bfc-a890b5150cb9)

Also keep in mind that the Sentry SDK requests were blocked by my AdBlocker so if you are running one be sure to check that it doesn't block the requests from the SDK 👍

**Reply by uhlhosting** [reply]
I usually always disable adblock and security allover on sites i work with. The issue was always that alto you seen the message maybe of replays sent. None are registered on sentry ever. The replays screen is always empty regardless of settings.

**Reply by stayallive** [reply, marked as answer]
Please double check with a sentry.io account temporary to make sure it's not a problem with your self-hosted instance! The plugin is sending the data so to me this seems like a problem with your self-hosted instance if there is still no data visible.
